editor/code_editor.cpp

```diff
@@ -1815,7 +1814,13 @@ CodeTextEditor::CodeTextEditor() {
 	status_bar->add_child(zoom_button);
 	zoom_button->set_flat(true);
 	zoom_button->set_v_size_flags(SIZE_EXPAND | SIZE_SHRINK_CENTER);
-	zoom_button->set_tooltip_text(TTR("Zoom factor"));
+#ifdef MACOS_ENABLED
+	const String wheel_zoom_modifier = TTR("Cmd");
+#else
+	const String wheel_zoom_modifier = TTR("Ctrl");
+#endif
+	zoom_button->set_tooltip_text(
+			TTR("Zoom factor") + "\n" + vformat(TTR("%s+Mouse wheel, %s/%s: Finetune"), wheel_zoom_modifier, ED_GET_SHORTCUT("script_editor/zoom_in")->get_as_text(), ED_GET_SHORTCUT("script_editor/zoom_out")->get_as_text()));
```

Comment:
```suggestion
	zoom_button->set_tooltip_text(
			TTR("Zoom factor") + "\n" + vformat(TTR("%sMouse wheel, %s/%s: Finetune"), keycode_get_string((Key)KeyModifierMask::CMD_OR_CTRL), ED_GET_SHORTCUT("script_editor/zoom_in")->get_as_text(), ED_GET_SHORTCUT("script_editor/zoom_out")->get_as_text()));
```

`keycode_get_string((Key)KeyModifierMask::CMD_OR_CTRL)` is used other places for similar tooltips.